apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.html.ts

```diff
@@ -333,6 +336,29 @@ export function InstructorFileEditor({
           </div>
         </div>
       </form>
+
+      ${SaveConfirmationModal()}
+    `,
+  });
+}
+
+function SaveConfirmationModal() {
+  return Modal({
+    title: 'Confirm Save',
+    id: 'save-confirmation-modal',
+    body: html`
+      <p><strong>Warning:</strong> The following issues were detected:</p>
+      <ul id="save-confirmation-issues" class="text-danger"></ul>
+      <p>
+        If you click <strong>Confirm Save</strong>, you may need to undo this change manually using
+        Git.
```

Comment:
Discussed on Slack, consider language like the following instead:

> If you save this change, it may introduce an error that can only be resolved through manual Git operations.